main.go

```diff
@@ -22,7 +22,13 @@ type Params struct {
 	Username      *string  `json:"username,omitempty"`
 }
 
+var (
+	buildDate string
```

Comment:
I don't think there is a right or wrong answer here. If you look at Docker they do compile this sort of information into their binary so there is some precedence for doing so. Since we do this for Drone core and many of the plugins I'm inclined to suggest we keep it.

```
Client:
 Version:      1.8.3
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   f4bf5c7
 Built:        Mon Oct 12 18:01:15 UTC 2015
 OS/Arch:      darwin/amd64
```